.env.sample

```
@@ -8,4 +8,8 @@ export NEXT_PUBLIC_HOST_URL="http://localhost:3000"
 export NEXT_PUBLIC_GA_ID=""
 
 # Google Earth Engine private key (e.g., {"type": "service_account","project_id": "ee-ocaufcg","private_key_id"})
-export NEXT_PUBLIC_GEE_PRIVATE_KEY=""
\ No newline at end of file
+export NEXT_PUBLIC_GEE_PRIVATE_KEY=""
+
+# Contentful credencials
+NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN=""
+NEXT_PUBLIC_CONTENTFUL_SPACE=""
```

I guess its nice add the export in the .env file

```suggestion
export NEXT_PUBLIC_CONTENTFUL_ACCESS_TOKEN=""
export NEXT_PUBLIC_CONTENTFUL_SPACE=""
```